### DISCUSSION TITLE
How do we buy a kit?

### DISCUSSION THREAD

**Discussion by rmurray484** [thehighnibble/thehighnibble.github.io#162] [Cromemco Z-1 replica]
Hello,

I want to buy a kit, either a Cromemco Z-1 or an IMSAI 8080 with an S-132 module. I tried sending emails to order a kit but received no response. I wanted to start a small club here and create a BBS with it. Some friends are interested. However, I haven't received any replies. Does the project still exist? If so, how can we get a kit or get on a waiting list for one?

Thank you for your help.

**Comment by pfcushing** [comment]
Did you read the "Expression of Interest" notes on the overview page(s)?  They are pretty clear on how interest is handled.
